tests/testing_data/inference.json

```diff
@@ -63,12 +63,12 @@
     "inferer": {
         "_target_": "SlidingWindowInferer",
         "roi_size": [
-            96,
-            96,
-            96
+            64,
```

Comment:
Could you please help change to use the spatial_shape property of `metadata.json`?

Thanks.

Reply:
sorry I don't get this one, can you provide more details?

Reply:
thanks I'll address this in https://github.com/Project-MONAI/MONAI/issues/3940 by reviewing the meta info.